pyro/nn/auto_reg_nn.py

```diff
@@ -110,8 +108,12 @@ class AutoRegressiveNN(nn.Module):
 
     :param input_dim: the dimensionality of the input
     :type input_dim: int
-    :param hidden_dim: the dimensionality of the hidden units
-    :type hidden_dim: int
+    :param hidden_dims: the dimensionality of the hidden units per layer
+    :type hidden_dims: list[int]
+    :param count_params: number of parameters to return from a forward call of the network.
+    :type count_params: int
+    :param param_dim: the dimensionality of each parameter
+    :type param_dim: int
     :param output_dim_multiplier: the dimensionality of the output is given by input_dim x output_dim_multiplier.
```

Comment:
isn't `output_dim_multiplier` a function of the other parameters?

Reply:
Ah sorry, I hadn't removed some defunct parameters from the docstring